storage/innobase/xtrabackup/doc/source/prerequisites/configuring.rst

```diff
@@ -0,0 +1,43 @@
+.. _configuring:
+
+Configuring XtraBackup
+======================
+
+Setting options configures the *Percona XtraBackup*. These options behave
+exactly like the standard |MySQL| program options: they can be specified either at
+the command-line or through a configuration file like `/etc/my.cnf`.
+
+See the :ref:`option and variable reference
+<xbk_option_reference>` for details on all of the configuration options.
+
+The *Percona XtraBackup* binary reads the ``[mysqld]`` and ``[xtrabackup]`` sections from any configuration files, in that order. This order allows the binary to read options from your existing |MySQL| installation, such as the `datadir` or certain *InnoDB* options. Specify any options to be overridden in the ``[xtrabackup]`` section. This section takes precedence since it is read later.
+
+The *Percona XtraBackup* binary does not accept exactly the same syntax in the
+`my.cnf` file as the `mysqld` server binary. For historical
+reasons, the `mysqld` server binary accepts parameters with a
+``--set-variable=<variable>=<value>`` syntax, which *Percona XtraBackup* does not understand. If your `my.cnf` file has these configuration directives rewrite them in the ``--variable=value`` syntax.
+
+You are not required to put any options in your `my.cnf`. You can simply specify the options on the command-line. 
+
+The ``target_dir`` option may be convenient to put in the ``[xtrabackup]`` section
```

Comment:
I would change to:
You can put the ``target_dir`` option into the ``[xtrabackup]`` section of your `my.cnf` file.

Reply:
Rewritten.